### ISSUE TITLE
repeating properties

### ISSUE THREAD

**Issue by thoffeller** [CAFM-Connect/BIM-Profiles#4]
Hi, is it in general possible to avoid the duplication of similar Propertytemplates for multiple IfcClassificationReferences? f.e. 3__Qv7dHH1WfjshtDSwWC$ has 6 instances. So, one set that is referenced by more than one IfcClassificationReference is more suitable?

**Comment by klacol** [comment]
Yes, that was a problem from the beginning, In our property database, we have now normalized the repeating properties. There are to open things on that:

1. We have to decide, which if them should be "remaining property" and what to do with the GUID's of the other propeties. We could just delete them. 
2. What, if a enum property has the same name, but different enum values? We then treat them as different properties,

That is the actual plan.

**Comment by thoffeller** [comment]
If 2. happens, it has to be a new property with a new name?

**Comment by klacol** [comment]
> If 2. happens, it has to be a new property with a new name?

Yes, but it can happen, that the name is the same. It is clear, that each property has an individual GUID. We should agree, that the name somehow is also different, to avoid unwanted conflicts between them.
